docs/proposals/job.md

```diff
@@ -0,0 +1,135 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller - which will be responsible
+for managing pod(s) that require to run-once to a completion even if the machine
```

Comment:
Nit, this sentence reads a little awkward -- 'require running once to completion', perhaps?